lib/db/schema/patch-072-073.sql

```diff
@@ -0,0 +1,239 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+-- Add columns `verified` and `enable` to help manage TOTP token state
+ALTER TABLE `totp`
+ADD COLUMN `verified` TINYINT(1) DEFAULT FALSE,
+ADD COLUMN `enable` TINYINT(1) DEFAULT TRUE,
```

Comment:
...yeah (he said, continuing the conversation with himself), I think `enabled` would be better actually, it seems more consistent with other column names.